builder/virtualbox/common/export_config.go

```diff
@@ -31,6 +32,19 @@ type ExportConfig struct {
 	// }
 	// ```
 	//
+	// In HCL2:
+	// ```hcl
+	//	source "virtualbox-iso" "basic-example" {
+	//		export_opts = [
+	//	      "--manifest",
+	//	      "--vsys", "0",
+	//	      "--description", "{{user `vm_description`}}",
+	//	      "--version", "{{user `vm_version`}}"
+	//	   	]
+	//		format = "ova"
+	//  }
```

Comment:
Not sure if this is github but it looks like the indentation is a bit off:
```suggestion
	//		export_opts = [
	//	          "--manifest",
	//	          "--vsys", "0",
	//	          "--description", "{{user `vm_description`}}",
	//	          "--version", "{{user `vm_version`}}"
	//	   	]
	//		format = "ova"
	//  }
```